pyproject.toml

```diff
@@ -16,6 +16,7 @@
 build-backend = "setuptools.build_meta"
 requires = [
     "cython>=3.0.0",
+    "libucx==1.15.0",
```

Comment:
Proposing here to build against a specific version (`==1.15.0`) but run against a range of compatible versions (`>=1.15.0,<1.16`), based on the pattern I saw in https://github.com/rapidsai/ucxx/pull/226.

Reply:
Yup that is exactly right, it's what we want.